packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -30,300 +34,405 @@ export default class Reachability extends StatelessWebexPlugin {
+   * @returns {undefined}
+   * @private
+   * @memberof Reachability
+   */
+  logUnreachableClusters() {
+    const list = this.getUnreachablClusters();
+
+    list.forEach((cluster) => {
+      LoggerProxy.logger.error(`Reachability:index#getLocalSDPForClusters --> No ice candidate for ${cluster}.`);
+    });
+  }
+
+
+  /**
+   * Calculates time to establish connection
+   * @param {array} iceResults iceResults
+   * @returns {object} reachabilityMap
+   * @private
+   * @memberof Reachability
+   */
+  parseIceResultsToReachabilityResults(iceResults) {
+    const reachabilityMap = {};
+
+    iceResults.forEach(({clusterId, elapsed}) => {
+      let latencyResult;
+
+      if (elapsed === null) {
+        latencyResult = {reachable: 'false'};
+      }
+      else {
+        latencyResult = {
+          reachable: 'true',
+          latencyInMilliseconds: (elapsed).toString()
+        };
+      }
+
+      reachabilityMap[clusterId] = {
+        udp: latencyResult,
+        tcp: latencyResult
       };
-    }));
-  };
+    });
+
+    return reachabilityMap;
+  }
+
+
+  /**
+   * fetches reachability data
+   * @param {object} clusterList
+   * @returns {Promise<localSDPData>} reachability check results
+   * @private
+   * @memberof Reachability
+   */
+  performReachabilityCheck(clusterList) {
+    if (!clusterList || !Object.keys(clusterList).length) {
+      return Promise.resolve({});
+    }
+
+    return new Promise((resolve) => {
+      this.getLocalSDPForClusters(clusterList)
+        .then((localSDPData) => {
+          if (!localSDPData || !Object.keys(localSDPData).length) {
+            // TODO: handle the error condition properly and try retry
+            LoggerProxy.logger.info('Reachability:index#performReachabilityCheck --> Local SDP is empty or has missing elements..returning');
+            resolve({});
+          }
+          else {
+            resolve(localSDPData);
+          }
+        })
+        .catch((error) => {
+          LoggerProxy.logger.error(`Reachability:index#performReachabilityCheck --> Error in getLocalSDPForClusters: ${error}`);
+          resolve({});
+        });
+    });
+  }
+
+
+  /**
+   * Records latency and closes the peerConnection
+   * @param {RTCPeerConnection} peerConnection
+   * @param {number} elapsed Latency in milliseconds
+   * @returns {undefined}
+   * @private
+   * @memberof Reachability
+   */
+  setLatencyAndClose(peerConnection, elapsed) {
+    const REACHABLE = 'reachable';
+    const UNREACHABLE = 'unreachable';
+    const {key} = peerConnection;
+    const resultKey = elapsed === null ? UNREACHABLE : REACHABLE;
+    const intialState = {[REACHABLE]: 0, [UNREACHABLE]: 0};
+
+    this.clusterLatencyResults[key] = this.clusterLatencyResults[key] || intialState;
+    this.clusterLatencyResults[key][resultKey] += 1;
+
+    // Set to null in case this fired from
+    // an event other than onIceCandidate
+    peerConnection.onicecandidate = null;
+    peerConnection.close();
```

Comment:
![Screen Shot 2020-11-17 at 5 25 02 PM](https://user-images.githubusercontent.com/14226615/99470601-d591c500-28f9-11eb-9f68-4762d3a314db.png)

guessing it comes from here.

Reply:
@akoushke - Thank you for sharing this, I will update the code. Mind sharing the browser/version?